tests/README.md

```diff
@@ -72,7 +72,7 @@ for this script.
 
 Note: this test cannot be run on Windows
 systems unless you are using Windows Subsystem for Linux.
-It also requires a Python version < 3.11 as pytype does not yet support
+It can currently only be run on Python 3.10 as pytype does not yet support
```

Comment:
We could avoid this if we added the `importlib_metadata` backport as a test dependency

Reply:
I prefer the narrow requirements in this case. I would be surprised if pytype_test is much run manually and an extra requirement can always be a bit of a pain.

Reply:
I agree, though I don't have a strong opinion at all